packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeRenderer.tsx

```diff
@@ -50,18 +53,20 @@ export function TreeRenderer({
   const nodeRenderer = useCallback<TreeProps["nodeRenderer"]>(
     (nodeProps) => {
       return (
-        <TreeNodeRenderer
-          {...nodeProps}
-          expandNode={expandNode}
-          getHierarchyLevelDetails={getHierarchyLevelDetails}
-          onFilterClick={onFilterClick}
-          onNodeClick={onNodeClick}
-          onNodeKeyDown={onNodeKeyDown}
-          getIcon={getIcon}
-        />
+        <LocalizationContextProvider localization={localization}>
```

Comment:
No need to wrap each node in context. You can wrap `<Tree<RendererdTreeNode> /> instead.

Reply:
Right, it was not supposed to wrap each node, fixed.